spec/features/peep_list_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'pg'
+
+feature 'Viewing peeps' do
+  scenario 'A user can see peeps' do
+    PG.connect(dbname: 'chitter_test')
```

Comment:
I'm a bit confused with this as I think your database connection does this already?